src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/SemanticTokensRangeParamsBridge.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using MediatR;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using VSSemanticTokensRangeParams = Microsoft.VisualStudio.LanguageServer.Protocol.SemanticTokensRangeParams;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts
+{
+    /// <summary>
+    /// This class is used as a "bridge" between the O# and VS worlds. Ultimately it only exists because the base <see cref="VSSemanticTokensRangeParams"/>
+    /// type does not implement <see cref="IRequest{TResponse}"/>.
+    /// </summary>
+    internal class SemanticTokensRangeParamsBridge : VSSemanticTokensRangeParams, IRequest<SemanticTokens?>
+    { }
```

Comment:
nit: separate lines ditto with `SemanticTokensRefreshParamsBridge`. Wonder if there's an editorconfig option for this